src/org/traccar/protocol/Jt600Protocol.java

```diff
@@ -17,15 +17,21 @@
 
 import org.jboss.netty.bootstrap.ServerBootstrap;
 import org.jboss.netty.channel.ChannelPipeline;
+import org.jboss.netty.handler.codec.string.StringEncoder;
 import org.traccar.BaseProtocol;
 import org.traccar.TrackerServer;
+import org.traccar.model.Command;
 
 import java.util.List;
 
 public class Jt600Protocol extends BaseProtocol {
 
     public Jt600Protocol() {
         super("jt600");
+        setSupportedCommands(Command.TYPE_ENGINE_RESUME,
```

Comment:
Well, I think I have updated formatting to match other protocol classes. Please check.